package.json

```diff
@@ -13,14 +13,14 @@
   "dependencies": {
     "@astro-community/astro-embed-youtube": "^0.5.3",
     "@astrojs/check": "^0.9.4",
-    "@astrojs/markdoc": "^0.12.10",
-    "@astrojs/mdx": "^4.1.0",
-    "@astrojs/netlify": "^6.2.2",
-    "@astrojs/react": "^4.2.1",
-    "@astrojs/sitemap": "^3.2.1",
-    "@astrojs/starlight": "^0.32.2",
-    "@astrojs/starlight-tailwind": "^3.0.0",
-    "@astrojs/tailwind": "^5.1.5",
+    "@astrojs/markdoc": "^0.13.3",
+    "@astrojs/mdx": "^4.2.3",
+    "@astrojs/netlify": "^6.2.5",
+    "@astrojs/react": "^4.2.3",
+    "@astrojs/sitemap": "^3.3.0",
+    "@astrojs/starlight": "^0.32.5",
+    "@astrojs/starlight-tailwind": "^3.0.1",
+    "@astrojs/tailwind": "^6.0.2",
```

Comment:
have you tested this? everything tailwind-update related needs to be verified because some packages still dont support the latest version so we can not update (starlight for example)